mflowgen/glb_top/construct.py

```diff
@@ -263,6 +263,10 @@ def construct():
   order.insert ( reporting_idx, 'dont-touch.tcl' )
   init.update_params( { 'order': order } )
 
+  # Update rtl node parameters
+  rtl.update_params( { 'num_glb_tiles': parameters['num_glb_tiles'] }, allow_new=True  )
+  rtl.update_params( { 'cgra_width': parameters['cgra_width'] }, allow_new=True  )
+
```

Comment:
Can we just add these parameters directly to the rtl node, rather than adding them via this construct.py?

Reply:
ok. I will remove these parameters from `construct.py`